man/man8/zfs.8

```diff
@@ -3500,7 +3500,10 @@ set when the stream is received.
 If the
 .Fl F
 flag is specified when this stream is received, snapshots and file systems that
-do not exist on the sending side are destroyed.
+do not exist on the sending side are destroyed. If this flag is used to send
+encrypted datasets, this flag requires that
+.Fl w
+is also used.
```

Comment:
> this flag requires that.Fl w is also used.

I found this last bit a little ambiguous.  Does this mean that `-w` will be set implicitly by `-R`, or that the caller must explicitly use `-Rw` when there encrypted dataset or it will fail.

Reply:
i didnt want anything to be done implicitly when it came to dataset security, so as of nor it requires that you add it. Doing `-R` alone will spit out an error. Do you have a better way to word that?

Reply:
How about, "If this flag is used to send encrypted datasets, then .Fl w must also be specified."

Reply:
@tcaputi Thank you for making these changes!  I have one minor request.  Instead of referring to "this" flag in the last sentence, can we say, "If the .Fl -R flag is used to send encrypted datasets, then .Fl -w flag must also be specified."  I am requesting this change as the use of "this" is ambiguous in the context of the two preceding sentences.  The immediately preceding sentence discusses the use of the `-F` flag with a `zfs receive` while the first sentence in the paragraph discusses using the `-i` and `-I` flags.  Thus, it may not be clear to which flag "this" is intended to refer.